pages/api/github.js

```diff
@@ -6,23 +6,32 @@ const getContributors = async (req, res) => {
       baseUrl: 'https://api.github.com',
     });
 
-    octokit.rest.repos
-      .listContributors({
+    const getUsersList = async repo => {
```

Comment:
Create `getUsersList` outside `getContributors`

Reply:
>we'll have access to the octokit variable

Why not move octokit declaration inside `getUsersList` as well?

> this function will be used anywhere else

This is right but, right now you are declaring a function inside a function which means that it will get declared every time the parent function is called, we certainly do not want that

Reply:
> > we'll have access to the octokit variable
> 
> Why not move octokit declaration inside `getUsersList` as well?
> 
> > this function will be used anywhere else
> 
> This is right but, right now you are declaring a function inside a function which means that it will get declared every time the parent function is called, we certainly do not want that

I've made the necessary changes. Thank you for the explanation.